java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture4/tasks/_6/Number6.java

```diff
@@ -0,0 +1,101 @@
+package com.zoolatech.lecture4.tasks._6;
+
+/*
+Create a method that accepts a string, counts number of letters (assume only English lower and upper letters),
+digits, spaces and other characters and outputs this data:
+Input: “aB?C de 42* dcd”
+Output:
+	“8 letters”
+	“2 digits”
+	“3 spaces”
+	“2 other characters”
+ */
+
+public class Number6 {
+    public static void main(String[] args) {
+        String string = "aB?C de 42* dcd";
+        VariationOne variationOne = new VariationOne();
+        variationOne.countStringSymbolsVariationOne(string);
+        VariationTwo variationTwo = new VariationTwo();
+        variationTwo.countStringSymbolsVariationTwo(string);
+    }
+}
+
+class VariationOne {
+
+    public void countStringSymbolsVariationOne(String string) {
+        int counterOfChars = 0;
+        int counterOfDigits = 0;
+        int couterForSpaces = 0;
+        int counterForOtherCharacters = 0;
+
+        for (char c = 'a'; c <= 'z'; c++) {
+            for (int i = 0; i <= string.length() - 1; i++) {
```

Comment:
`i <= string.length() - 1` can be changed to `i < string.length()` which looks simpler